loc/items/gem.js

```diff
@@ -2,76 +2,77 @@ export default [
   {
     name       : {en: 'Lucid' },
     description: {
-      en: 'Gem description'
+      en: 'The sharp green edges catch sparkling light in the sun, making a beautiful and deadly sight.'
     }
   },
 
   {
     name       : {en: 'Melant' },
     description: {
-      en: 'Gem description'
+      en: 'Rough disc-shaped gray stone with strange druidical engravings encasing it.'
     }
   },
 
   {
     name       : {en: 'Turim' },
     description: {
-      en: 'Gem description'
+      en: ''
     }
   },
 
   {
     name       : {en: 'Fundo' },
     description: {
-      en: 'Gem description'
+      en: 'Brown desaturated gem, emitting putrid scents as you wear it around your neck.'
     }
   },
 
   {
     name       : {en: 'Royal' },
     description: {
-      en: 'Gem description'
+      en: 'Grand red gem enclosed in abyss dark fissures, granting your foes a sense of jealousy.'
```

Comment:
Grand red, abyss dark is a bit too epic for this tier I think. but the general idea is nice.